defaults/main.yml

```diff
@@ -1,2 +1,2 @@
 ---
-terraform_version: "0.6.12"
+terraform_version: "0.6.12"
```

Comment:
In most cases, the consuming projects override `terraform_version` at the top level, so I wouldn't worry about it. Also, including this change in the `CHANGELOG` will give it good visibility.